src/bin/pg_dump/dumputils.c

```diff
@@ -1062,7 +1062,7 @@ char *
 custom_fmtopts_string(const char *src)
 {
 		int			len = src ? strlen(src) : 0;
-		char	   *result = malloc(len * 2 + 1);
+		char	   *result = calloc(1, len * 2 + 1);
```

Comment:
I think this is a weird usage, for the usage of calloc, the first parameter should be count, and the second should be size. What about call(len*2+1, sizeof(char)) ?